core/test/ParaGrid_test.cpp

```diff
@@ -161,18 +207,45 @@ TEST_CASE("Write and read a ModelState-based ParaGrid restart file")
     metadata.extractCoordinates(coordState);
     metadata.affixCoordinates(state);
 
+#ifdef USE_MPI
+    metadata.setMpiMetadata(test_comm);
+    if (metadata.mpiMyRank == 0) {
+        metadata.setMpiMetadata(test_comm);
+        metadata.globalExtentX = nx;
+        metadata.globalExtentY = ny;
+        metadata.localCornerX = start_x;
+        metadata.localCornerY = 0;
+        metadata.localExtentX = local_nx;
+        metadata.localExtentY = ny;
+    }
+    if (metadata.mpiMyRank == 1) {
+        metadata.setMpiMetadata(test_comm);
+        metadata.globalExtentX = local_nx;
```

Comment:
Is this right?

Reply:
thanks for spotting this. It's been fixed :+1: